pkg/minikube/download/preload.go

```diff
@@ -40,7 +40,7 @@ const (
 	// PreloadVersion is the current version of the preloaded tarball
 	//
 	// NOTE: You may need to bump this version up when upgrading auxiliary docker images
-	PreloadVersion = "v3"
+	PreloadVersion = "v4"
```

Comment:
if you do this in a separate PR we chould make the preload images and then test this PR with the generated prelaoded imges.

Reply:
I've updated the v4 images - would you prefer that the addition of kindnet as a CNI be split out into a separate PR?